pr-data.csv

```diff
@@ -3831,7 +3831,7 @@ https://github.com/espertechinc/esper,50eddcdc2e7894de20309cd0b6f585110e4bc61b,c
 https://github.com/espertechinc/esper,50eddcdc2e7894de20309cd0b6f585110e4bc61b,common,com.espertech.esper.common.client.configuration.TestConfigurationParser.testRegressionFileConfig,ID,Accepted,https://github.com/espertechinc/esper/pull/227,commit:https://github.com/espertechinc/esper/commit/cf8535af39ad50fd9cbb53cd90904796218039c8
 https://github.com/espertechinc/esper,50eddcdc2e7894de20309cd0b6f585110e4bc61b,common,com.espertech.esper.common.internal.epl.join.assemble.TestLeafAssemblyNode.testProcess,ID,Accepted,https://github.com/espertechinc/esper/pull/226,commit:https://github.com/espertechinc/esper/commit/eec1489df9a9c26b5ee27533ddaef48effa0734e
 https://github.com/espertechinc/esper,590fa9c9eb854f1420b9d337b802aca19f963cc0,examples/rfidassetzone,com.espertech.esper.example.rfidassetzone.TestLRMovingSimMain.testSim,NDOD;NOD,,,
-https://github.com/espertechinc/esper,590fa9c9eb854f1420b9d337b802aca19f963cc0,regression-run,com.espertech.esper.regressionrun.suite.client.TestSuiteClientCompile.testClientCompileEventTypeAutoName,ID,,,
+https://github.com/espertechinc/esper,590fa9c9eb854f1420b9d337b802aca19f963cc0,regression-run,com.espertech.esper.regressionrun.suite.client.TestSuiteClientCompile.testClientCompileEventTypeAutoName,ID,DeveloperFixed,https://github.com/espertechinc/esper/commit/eec1489df9a9c26b5ee27533ddaef48effa0734e,
```

Comment:
Same as previous -- wrong column. The second last column is always for the PR opened by us (so usually `Opened`, `Accepted`, `Rejected` statuses). The last column is for notes (so all other statuses).